test/SynthsSdk.test.ts

```diff
@@ -49,9 +54,36 @@ describe("Synths SDKs", function () {
       upunksAsset = synthsSDK.connectAsset("upunks-0921");
     });
 
-    // @todo Add tests
     describe("Interact with asset", function () {
+      it("helpers - success", async function () {
+        this.timeout(100000);
+        const synthPrice = await getCurrentDexTokenPrice(
+          "sushiswap",
+          "0x6e01db46b183593374a49c0025e42c4bb7ee3ffa",
+          "0x86140A763077155964754968B6F6e243fE809cBe"
+        );
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const synthData = await getSynthData(
+          "upunks-0921",
+          "0x86140A763077155964754968B6F6e243fE809cBe"
+        );
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const synthChartData = await getSynthChartData(
+          "0x86140A763077155964754968B6F6e243fE809cBe"
+        );
+        const response = await axios.get(
+          `https://data.yam.finance/degenerative/apr/upunks-0921`
+        );
+
+        expect(synthData).to.deep.include({
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+          apr: response.data["aprMultiplier"] as string,
+        });
+        expect(synthChartData).to.be.an("array");
+        expect(synthPrice).to.not.equal(0);
+      });
```

Comment:
for now thats ok but as a next step keep in mind to separate all the helper functions each individually, to eventually become something like that:
- ...
- Interact with asset
  - Asset helpers
    - `getCurrentDexTokenPrice - success`
    - `getSynthData- success`
    - `getSynthChartData- success`
    - etc.
- ...

Reply:
Yes, I will update these  tests soon in another pr.